pkg/apis/apis/v1alpha1/types.go

```diff
@@ -378,6 +384,14 @@ type APIExportPolicy struct {
 // of the API Export
 type LocalAPIExportPolicy struct{}
 
+type PermissionClaim struct {
+	metav1.GroupResource `json:","`
```

Comment:
does this give us a good schema? I think last time I checked there was nothing specified about the group and resource strings. So we went with a custom struct.

Reply:
check scheduling api group, GroupVersionResource. You can steal the speak from there.

Reply:
+1 didn't think about this thanks!

Reply:
updated